README.md

```diff
@@ -7,32 +7,31 @@
 [![Docker image](https://img.shields.io/docker/pulls/vshn/k8up)][dockerhub]
 [![License](https://img.shields.io/github/license/vshn/k8up)][license]
 
-<img src="https://raw.githubusercontent.com/vshn/k8up/master/docs/images/logo.png" width="150">
+<img src="https://raw.githubusercontent.com/vshn/k8up/master/docs/modules/ROOT/assets/images/k8up-logo.png" width="150">
```

Comment:
What would it look like if the SVG is referenced here?
Also, in MD images can be referenced like this:
`![alt text](<url> "Logo Title Text 1")`